style.js

```diff
@@ -0,0 +1,33 @@
+const navToggler = document.querySelector(".myNav-toggler");
+navToggler.addEventListener('click', () => {
+   hideSection();
+   toggleNavbar()
+    document.body.classList.toggle("hide-scrolling")
+});
+
+function hideSection(){
+    document.querySelector("section.active").classList.toggle("fade-out")
```

Comment:
This is a difficult way to do this. Let me break this down the way I would do it:

When I click on the menu, I want all the sections to be hidden.
When I click on a nav-item or the toggleNavbar button, I want all the sections to be revealed.

Everything else is handled for us in the HTML. Try pseudo-coding that; to do that you will have to change this function to toggle `fade-out` on EVERY section, not just the active one.

Reply:
I tried but when I did toggleNavbar button stops working. maybe I'm doing something wrong in that.  Can we solve this together on Monday?